backend/docs/buildingsV1.yml

```diff
@@ -118,16 +132,17 @@ components:
       properties:
         name:
           type: string
-        level:
+        floor_level:
```

Comment:
Should we have this attribute at all since this room will reside in a floor array? is this redundant?